steeplechase/runsteeplechase.py

```diff
@@ -499,7 +499,7 @@ def get_manifest(req):
     log.info("Result summary:")
     log.info("Passed: %d" % pass_count)
     log.info("Failed: %d" % fail_count)
-    return fail_count == 0
+    return (pass_count > 0 and fail_count == 0)
```

Comment:
I don't like the parens. Either parenthesize each side of the and, or drop them entirely.

Note that I am not the maintainer....